libr/bin/bin.c

```diff
@@ -752,32 +752,13 @@ R_API RList *r_bin_get_libs(RBin *bin) {
 	return o ? o->libs : NULL;
 }
 
-R_API RList *r_bin_patch_relocs(RBin *bin) {
+R_API RBNode *r_bin_patch_relocs(RBin *bin) {
```

Comment:
add r_bin_patch_relocs_list and make this function return bool. we already have the get_relocs() to get the rbnode

Reply:
no, as this function is used right now it needs to return a RList/RBNode. We may think of whether this is good or not in additional refactoring IMHO.